### PR TITLE
Add suffixes to typos from "dep" to "der"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -17148,35 +17208,71 @@ deplying->deploying
...
+depnding->depending
 depnds->depends
 deporarily->temporarily
 deposint->deposing
 depoy->deploy
 depoyed->deployed
 depoying->deploying
+depoyment->deployment
+depoyments->deployments
+depoys->deploys
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:17236]: No occurrences of `depots` or `decoys`? Same for singular.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -17187,15 +17283,28 @@ depricate->deprecate
...
 depricates->deprecates
 depricating->deprecating
+deprication->deprecation, deprivation,
 dequed->dequeued
 dequeing->dequeuing
 deques->dequeues
 derageable->dirigible
 deram->dram, dream,
-derective->directive
+derect->direct, detect, defect,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:17292]: Perhaps add `erect`? **D** and **E** are neighbours on many keyboards.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:17236]: I will add both.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -17187,15 +17283,28 @@ depricate->deprecate
...
-derective->directive
+derect->direct, detect, defect, erect,
+derected->directed, detected, defected, erected,
+derecting->directing, detecting, defecting, erecting,
+derection->direction, detection, erection,
+derections->directions, detections, erections,
+derective->directive, detective, defective,
+derectives->directives, detectives,
+derectly->directly
+derector->detector, director,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:17300]: We could add `erector` here, like in "erector muscle".

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**